apps/condo/domains/resident/access/SendResidentMessageService.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Generated by `createservice resident.SendResidentMessageService --type mutations`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+async function canSendResidentMessage (data) {
+    const { authentication: { item: user } } = data
+
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+    if (user.isSupport) return false
+    if (user.type === SERVICE) return true
```

Comment:
wtf?

Reply:
There will be cron task, either within condo or EPS, that will trigger this service on new EPS billing receipts arrival. If task will be on EPS side, it will use condoBot to make requests to condo, that uses service user for access.

Reply:
You need to check billing access. And message type.